SparkPost/2web2ui
# FE-1251 - Create Empty state for Snippets page and Add banner to Snippets page

FE-1251 - Create Empty state for Snippets page and Add banner to Snippets page

### What Changed

- Added Empty state for Snippets page 
- Added banner to Snippets Listpage
- Add the prop hibanaEmptyStateComponent to Page Wrapper
- Added the segmentTrack to Page 
- Removed useEmptyState from api keys and abtest page and used the hibanaEmptyStateComponent prop instead
- Removed useEmptyState

### How To Test
 - Enable account ui flag - "allow_empty_states"
 - When Snippets = 0 and hibana is enabled, verify you see the Empty state.
 - verify that a event is sent to segment when empty state loads
 - When Snippets > 0 and hibana is enabled, verify you see the Banner.
      - Verify once the banner is dismissed it doesn't show up again
 - Check if api keys Empty state and ab test empty state render correctly.
   

### To Do

- [x] Fix the failing unit test
- [x] Remove useEmptyState hook and fix it in AB test and Api keys
